src/assets/i18n/en.json

```diff
@@ -84,6 +85,7 @@
   "login.create.assignment.description": "You need to create a profile before you can create a job",
   "login.apply.assignment.title": "Search jobs",
   "login.apply.assignment.description": "You need to create a profile before you can search for a job",
+  "start.or": "or",
```

Comment:
Looks like this is because we want to say `%value% or %other_value%`. This wont work for arabic since the "pattern" of the phrase could perhaps be different in certain circumstances. 

For example `{{value} or {{other_value}}` :) See the [`ng-translate` docs](https://github.com/ocombe/ng2-translate#3-define-the-translations) about `TranslateParser` :)